app/assets/stylesheets/thredded/components/_topic-header.scss

```diff
@@ -60,3 +60,67 @@
   font-size: $thredded-font-size-small;
   color: $thredded-secondary-text-color;
 }
+
+
+@mixin following {
+  &::after {
+    display: inline-block;
+    font-size: 1em;
+    width: 1.4rem;
+    height: 1.4rem;
+    position: absolute;
+    top: 0;
+    right: -1.6rem;
+    background-image: image-url("thredded/follow.svg");
```

Comment:
With non-inline SVGs there is no way to change the color from CSS. Does this look OK on all themes?